src/app/pages/features.html

```diff
@@ -142,16 +145,13 @@ <h2>ALLOW DEVELOPMENT TEAMS TO MAKE MORE CONFIDENT TECHNOLOGY DECISIONS</h2>
           <img src="../../assets/images/screencap_pipeline_complete.png" alt="Pipelines" width="100%" />
         </div>
         <div>
-          <h2>SIMPLE TO CREATE CONTAINERIZED DEV, TEST AND STAGING ENVIRONMENTS</h2>
-          <p>
-            Developers can automatically create containerized development environments with the workspace management capabilities of Eclipse Che.
-          </p>
+          <h2>CREATE CONTAINERIZED DEV, TEST, AND STAGE ENVIRONMENTS WITH EASE</h2>
           <p>
+            OpenShift.io creates environments hosted on OpenShift Online, providing the following advantages:
             <ul>
-              <li>OSIO creates environments hosted on OpenShift Online</li>
-              <li>Integrates the Jenkins Pipeline plugins</li>
-              <li>Create simplest ‘just build it’ pipeline or add promotions</li>
-              <li>Pipeline definitions written using a Groovy DSL</li>
+              <li>Integrated Jenkins Pipelines plugins</li>
+              <li>Flexibility to create simple “just build it” pipelines or add promotions</li>
```

Comment:
This one didn't make a lot of sense to me as it was edited.  How about "Flexibility to have simple automated build pipelines or add manual promotions into pipelines"

Reply:
This sounds good. I think I'd just like to avoid repeating pipelines. Does this work? -- "Flexibility to have simple automated build pipelines or add manual promotions"